tests/toolkit/timeseries_tests.cc

```diff
@@ -0,0 +1,27 @@
+#include <gtest/gtest.h>
+
+#include "agent.h"
+
+#include "../test_context.h"
+#include "../test_agents/test_agent.h"
+
+namespace cyclus {
+namespace toolkit {
+
+TEST(TimeSeriesTests, Power) {
+  TestContext tc;
+  Agent* a = new TestAgent(tc.get());
+  RecordTimeSeries<POWER>(a, 42.0);
+  delete a;
+}
+
+TEST(TimeSeriesTests, RawPower) {
+  TestContext tc;
+  Agent* a = new TestAgent(tc.get());
+  RecordTimeSeries<double>("Power", a, 42.0);
+  delete a;
+}
```

Comment:
While we shouldn't be (re)testing the database writing/querying here, I will note that these tests are not particularly useful - they test nothing about the functions except that they can be called without throwing exceptions or segfaulting - which IMO, is hardly better than having no tests at all w.r.t. how much tests can actually do.